### PR TITLE
PE-7006 add playwright mobile test

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: test-results/.last-run.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (dtfiedler) [inline test-results/.last-run.json:1]: Why do we need this?

Comment (atticusofsparta) [reply test-results/.last-run.json:1]: Dont think we do, can add to git ignore

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -21,6 +21,29 @@ jobs:
...
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-browsers-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            playwright-browsers-${{ runner.os }}-
+
+      # Install Playwright browsers if not cached
+      - run: yarn playwright install
+
+      - run: yarn build
```
Comment (dtfiedler) [inline .github/workflows/build_and_test.yml:43]: do we need to build?

File: .github/workflows/build_and_test.yml [hunk 3]
```diff
@@ -21,6 +21,29 @@ jobs:
...
+
+      - run: yarn ${{ matrix.step }}
```
Comment (dtfiedler) [inline .github/workflows/build_and_test.yml:45]:
```suggestion
      - run: yarn test:playwright
```

Comment (atticusofsparta) [reply .github/workflows/build_and_test.yml]: [2ff2601](https://github.com/ar-io/arns-react/pull/576/commits/2ff260110d4a7609c3067e0445652dffc66f08dc)

File: .github/workflows/build_and_test.yml (hunk 2 above)
Comment (atticusofsparta) [reply .github/workflows/build_and_test.yml:42]: Scratch that, we do need it https://github.com/ar-io/arns-react/commit/8c5e27d2ba8a7e8155700c899f529cb71bd67b62

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -16,6 +16,27 @@ jobs:
...
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-browsers-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            playwright-browsers-${{ runner.os }}-
+
+      # Install Playwright browsers if not cached
+      - run: yarn playwright install
+      - run: yarn build
+      - run: yarn test:playwright
```
Comment (dtfiedler) [inline .github/workflows/pr.yml:38]: nit: this whole file can just call the `build_and_test` workflow vs. having to duplicate. see an example of that here - https://github.com/ar-io/ar-io-sdk/blob/main/.github/workflows/release.yml#L14-L16

File: .github/workflows/pr.yml [hunk 5]
```diff
@@ -6,19 +6,12 @@ on:
...
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        step: ['lint:check', 'test', 'build']
-    steps:
-      - uses: actions/checkout@v3
-      - run: yarn --frozen-lockfile
-      - run: yarn ${{ matrix.step }}
-
+  build_ant_test:
```
Comment (dtfiedler) [inline .github/workflows/pr.yml:9]: nit
```suggestion
  build_and_test:
```

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**